frontend/src/utils/constants.tsx

```diff
@@ -117,6 +117,18 @@ export const MAX_TILES_SPEAKER_VIEW_MOBILE = 2;
 export const RIGHT_PANEL_BUTTON_COUNT = 3 - (isReportIssueEnabled() ? 0 : 1);
 
 /**
+ * @constant {number} CAPTION_TIMEOUT_MS - The duration in milliseconds for which captions are displayed.
+ * This is used to control how long the captions remain visible before they disappear.
```

Comment:
Nit: the latter comment is superfluous.  Reason being, you're just rephrasing the original comment, with no additional insight.

Reply:
fair enough. overexplaining myself isn't always the best 🙂